src/ucp/wireup/wireup_cm.c

```diff
@@ -370,12 +370,11 @@ ucp_wireup_cm_ep_cleanup(ucp_ep_t *ucp_ep, ucs_queue_head_t *queue)
         /* During discarding transfer the pending queues content from the
          * previously configured UCP EP to a temporary queue for further
          * replaying */
-        ucp_worker_discard_uct_ep(ucp_ep, ucp_ep_get_lane(ucp_ep, lane_idx),
-                                  ucp_ep_get_rsc_index(ucp_ep, lane_idx),
-                                  UCT_FLUSH_FLAG_CANCEL,
-                                  ucp_request_purge_enqueue_cb, queue,
-                                  (ucp_send_nbx_callback_t)ucs_empty_function,
-                                  NULL);
+        ucp_worker_discard_uct_ep(
```

Comment:
1. update the comment
2. remove "queue" parameter from ucp_wireup_cm_ep_cleanup - seems not needed anymore

Reply:
1. Will do
2. Will remove if it's indeed not needed.